css/desktop.css

```diff
@@ -0,0 +1,31 @@
+.hero-header {
+    display: grid;
+    place-items: center;
+    background-image: url('/img/hero-lg.jpg');
+    background-size: 100%;
+    background-repeat: no-repeat;
+}
+
+.hero-text {
+    width: 74rem;
+    padding: 22.5vw 0 0 0;
+}
+
+.star-logo {
+    max-width: 18rem;
+    margin-bottom: 2.5rem;
+}
+
+.description {
+    font-size: 4rem;
+    margin-bottom: 2.4rem;
+}
+
+.primary-button .quiero-combo{
```

Comment:
With [linear gradient](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient()), you can get the button's background-color

`background: linear-gradient(90deg,#fa1660,#ff6e00);`